src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/PublicationTests.cs

```diff
@@ -0,0 +1,160 @@
+using System;
+using System.Collections.Generic;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Tests
+{
+    public class PublicationTests
+    {
+        [Fact]
+        public void LatestPublishedRelease_NewDraftAmendment()
+        {
+            var originalReleaseId = Guid.NewGuid();
+            var amendedRelease1Id = Guid.NewGuid();
+            var amendedRelease2Id = Guid.NewGuid();
+            var publication = new Publication
+            {
+                Releases = new List<Release>
+                {
+                    new()
+                    {
+                        Id = originalReleaseId,
+                        Version = 0,
+                        ReleaseName = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        Published = DateTime.UtcNow.AddDays(-2),
+                        PreviousVersion = null,
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+
+                    },
+                    new()
+                    {
+                        Id = amendedRelease1Id,
+                        Version = 1,
+                        ReleaseName = "2000",
+                        Published = DateTime.UtcNow.AddDays(-1),
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        PreviousVersionId = originalReleaseId,
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                    },
+                    new()
+                    {
+                        Id = amendedRelease2Id,
+                        Version = 2,
+                        ReleaseName = "2000",
+                        Published = null,
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        PreviousVersionId = amendedRelease1Id,
+                        ApprovalStatus = ReleaseApprovalStatus.Draft,
+                    },
+                },
+            };
+
+            var result = publication.LatestPublishedRelease();
+            Assert.Equal(amendedRelease1Id, result.Id);
+        }
+
+        [Fact]
+        public void LatestPublishedRelease_NewDraft()
+        {
+            var release2000Id = Guid.NewGuid();
+            var release2001Id = Guid.NewGuid();
+            var release2002Id = Guid.NewGuid();
+            var publication = new Publication
+            {
+                Releases = new List<Release>
+                {
+                    new()
+                    {
+                        Id = release2000Id,
+                        Version = 0,
+                        ReleaseName = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        Published = DateTime.UtcNow.AddDays(-2),
```

Comment:
Published value doesn't matter here as long as they are published, and setting up one with -2 and one with -1 implies you might be expecting it to influence the order in some way. I'd be tempted to just use `DateTime.UtcNow` as it's the Year and TimePeriod and presence of the Published value which are important.